tlsr9/drivers/TL321X/usbhw.h

```diff
@@ -376,7 +380,7 @@ static inline void usbhw_set_eps_en(usb_ep_en_e ep)
     }
     else
     {
-        reg_usb_edp_en |= ep;
```

Comment:
zephyr must use this assignment method. @damien0x0023

Reply:
fixed.